automated/linux/mmtests/mmtests.yaml

```diff
@@ -54,7 +54,17 @@ params:
 run:
     steps:
         - cd ./automated/linux/mmtests/
-        - ./mmtests.sh -s "${SKIP_INSTALL}" -v "${TEST_PROG_VERSION}" -p "${TEST_DIR}" -u "${TEST_GIT_URL}" -c "${MMTESTS_CONFIG_FILE}" -t "${MMTESTS_TYPE_NAME}" -r ${MMTESTS_MAX_RETRIES}
+        - |
+            cmd="./mmtests.sh";
+            [ -n "${SKIP_INSTALL}" ] && cmd+=" -s ";
```

Comment:
'cmd+=" -s "' is failing in tuxrun, however, 'cmd="${cmd} -s "' works

Reply:
That's odd. What shell is used in tuxrun?

Reply:
Fixed.